### DISCUSSION TITLE
add auto-admin, plz

### DISCUSSION THREAD

**Discussion by zedd-0** [TheBobPony/MSEdgeTweaker#8] [Ideas]
`PowerShell.exe  " sc.exe pause rpcss > $null ; if ($LASTEXITCODE -eq 5) {Start-Process -Verb 'RunAs' '%~0' %* ; exit 0 } "  &&  goto :end0`

with this line, you can run your script into admin mode,
it will trigger the UAC.

**Comment by SmearODeer** [comment]
I can Do That, Adding to my Pull request now...
